건우/Week2_중복된숫자개수.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, n) {
+    var answer = 0;
```

Comment:
```suggestion
    let answer = 0;
```
modern JS에서는 var를 권장하지 않습니다. scope때문에 그러한데요
지금은 아직 scope에 대해서 배우지 않았으니 var 대신 const나 let을 써야한다는 사실 정도만 인지해놓으시면 좋을 것 같습니다.